packages/rocketchat-lib/server/methods/setUsername.coffee

```diff
@@ -11,7 +11,7 @@ Meteor.methods
 			return username
 
 		if not /^[0-9a-zA-Z-_.]+$/.test username
-			throw new Meteor.Error 'username-invalid'
+			throw new Meteor.Error 'username-invalid', TAPi18next.t('project:Username_invalid', username)
```

Comment:
Why `TAPi18next.t('project:Username_invalid', username)`?
The translation should not be done on the server side, as the client may be using a different language.